Model/lib/wdk/model/questions/queries/geneQueries.xml

```diff
@@ -1788,22 +1788,21 @@ AND 1=0
               SELECT distinct gs.transcript_source_id AS source_id, gs.gene_source_id, ta.project_id,
                     'Y' as matched_result,
                     apidb.tab_to_string(set(cast(COLLECT(distinct gs.evidence_code) AS apidb.varchartab)), ', ') as evidence_code,
---                  listagg(gs.is_go_slim, ',')WITHIN GROUP (ORDER BY gs.is_go_slim) as is_go_slim
                     regexp_replace(apidb.tab_to_string(set(cast(COLLECT(distinct to_char(gs.is_go_slim)) AS apidb.varchartab)), ', '),
                                      ', $', '')
                       as is_go_slim
               FROM ApidbTuning.TranscriptAttributes ta, ApidbTuning.GoTermSummary gs
               WHERE ta.taxon_id IN ($$organism$$)
                 AND ta.aa_sequence_id = gs.aa_sequence_id
                 AND decode(gs.evidence_code, 'IEA', 'Computed', 'Curated') in ($$go_term_evidence$$)
---              and case when $$go_term_slim$$ = 'Yes' and gs.is_go_slim = '1' then 1 
---                       when ($$go_term_slim$$ = 'No' and (gs.is_go_slim = '1' or gs.is_go_slim = '0')) then 1 
---                       else 0
```

Comment:
I see that these predicates related to GO slim, which were commented out, are removed. As we discussed, the commented-out version is kind of ungainly, but the whole thing could be replaced by "AND (dollar-dollar-go_term_slim-dollar-dollar = 'No' OR gs.is_go_slim = '1')", which not only is more readable (in my opinion), but also would perform better, since it wouldn't perform that CASE expression on every row. I don't have an opinion about whether users want that functionality, but I do think we should remove the GO slim param if it isn't going to do anything.